rules.neon

```diff
@@ -27,6 +27,7 @@ rules:
 	- PHPStan\Rules\DisallowedConstructs\DisallowedEmptyRule
 	- PHPStan\Rules\DisallowedConstructs\DisallowedImplicitArrayCreationRule
 	- PHPStan\Rules\DisallowedConstructs\DisallowedShortTernaryRule
+	- PHPStan\Rules\DisallowedConstructs\DisallowedWeakComparisonRule
```

Comment:
This rule has to be enabled only with bleedingEdge right now. Do a similar thing that was done here: https://github.com/phpstan/phpstan-doctrine/commit/00e7f8f97b2172793006e72da48714e881da6567

Reply:
Should also be fixed by https://github.com/phpstan/phpstan-strict-rules/pull/177/commits/56a7475d7185c829770fc9d1e721c5a3aaadceb1. But honestly, I don't know if I got it right. I am still lacking understanding of which configuration options are available, what they mean and how the interact with each other.